addons/hw_drivers/tools/helpers.py

```diff
@@ -341,6 +341,22 @@ def load_certificate():
         start_nginx_server()
     return True
 
+def delete_iot_handlers(iot_handlers_path):
```

Comment:
why recreate the wheel?

1 - list all file to remove => `list_file_by_os()`
2 - delete these files => `unlink_file()`

Bonus: Overrides from security not needed ! :)